cmd/tailscale/cli/status.go

```diff
@@ -127,6 +127,21 @@ func runStatus(ctx context.Context, args []string) error {
 
 	var buf bytes.Buffer
 	f := func(format string, a ...interface{}) { fmt.Fprintf(&buf, format, a...) }
+
+	if st.TailscaleIPs != nil {
+		f("Tailscale IPs: ")
```

Comment:
I think this can be made to look like the other lines status prints. That is, fit into a single line like:

[_____]       <tailscale IP>         <relay> <endpoints>

You can leave the fields you don't have as blank spaces right now (and always omit tx/rx, as those are always zero).